### PR TITLE
Fixing typos, spelling mistakes and missing text in EIP-1.

### PR DESCRIPTION
**Pull request by jonathancross**

### CODE REVIEW

File: EIPS/eip-1.mediawiki [hunk 1]
```diff
@@ -38,15 +38,15 @@ The EIP process begins with a new idea for Ethereum. It is highly recommended th
...
 
-If the EIP collaborators approves, the will assign the EIP a number, label it as Standards Track, Informational, or Process, give it status "Draft", and add it to the git repository. The EIP editor will not unreasonably deny a EIP. Reasons for denying EIP status include duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the Ethereum philosophy.
+If the EIP collaborators approves, the EIP editor will assign the EIP a number, label it as Standards Track, Informational, or Process, give it status "Draft", and add it to the git repository. The EIP editor will not unreasonably deny an EIP. Reasons for denying EIP status include duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the Ethereum philosophy.
```
Comment (jonathancross) [inline EIPS/eip-1.mediawiki]: FYI: Added "EIP editor" here.

I now see that there is another typo in the same line, but can make that fix in a future pull request.
eg: "If the EIP collaborators approve**s**" => "If the EIP collaborator(s) approve"

File: EIPS/eip-1.mediawiki [hunk 2]
```diff
@@ -62,7 +62,7 @@ EIPs can also be superseded by a different EIP, rendering the original obsolete.
 
 The possible paths of the status of EIPs are as follows:
 
-<img src=./eip-1/process.png></img>
```
Comment (jonathancross) [inline EIPS/eip-1.mediawiki]: FYI: `img` tag is empty (cannot have a closing tag)

File: EIPS/eip-1.mediawiki [hunk 3]
```diff
@@ -168,8 +168,8 @@ Once the EIP is ready for the repository, the EIP editor will:
 
 Many EIPs are written and maintained by developers with write access to the Ethereum codebase. The EIP editors monitor EIP changes, and correct any structure, grammar, spelling, or markup mistakes we see.
 
-The editors don't pass judgement on EIPs. We merely do the administrative & editorial part. Except for times like this, there's relatively low volume.
+The editors don't pass judgment on EIPs. We merely do the administrative & editorial part. Except for times like this, there's relatively low volume.
 
 ==History==
 
-This document was derived heavily from [https://github.com/bitcoin/bips Bitcoin's BIP-0001] written by Amir Taaki wich in turn was derived from [https://www.python.org/dev/peps/ Python's PEP-0001]. In many places text was simply copied and modified. Although the PEP-0001 text was written by Barry Warsaw, Jeremy Hylton, and David Goodger, they are not responsible for its use in the Ethereum Improvement Process, and should not be bothered with technical questions specific to Ethereum or the EIP. Please direct all comments to the EIP editors.
```
Comment (jonathancross) [inline EIPS/eip-1.mediawiki]: FYI: "wich" => "which"

File: EIPS/eip-1.mediawiki [hunk 4]
```diff
@@ -38,15 +38,15 @@ The EIP process begins with a new idea for Ethereum. It is highly recommended th
...
 
-Vetting an idea publicly before going as far as writing a EIP is meant to save the potential author time. Asking the Ethereum community first if an idea is original helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussions (searching the internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used.
+Vetting an idea publicly before going as far as writing a EIP is meant to save the potential author time. Asking the Ethereum community first if an idea is original helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussions (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used.
```
Comment (jonathancross) [inline EIPS/eip-1.mediawiki]: FYI: Capitalized "internet"

**Comment by wanderer** [comment]
looks good!

**Comment by jonathancross** [comment]
Cheers.  :-)

**Pull request merged**